packages/eslint-plugin/tests/schemas.test.ts

```diff
@@ -0,0 +1,91 @@
+import Ajv from 'ajv';
+import type { JSONSchema4 } from 'json-schema';
+
+import eslintPlugin from '../src';
+import { RuleModule } from '@typescript-eslint/utils/src/ts-eslint';
+
+describe("Validating rule's schemas", () => {
+  // These two have defaults which cover multiple arguments that are incompatible
+  const excluded = ['semi', 'func-call-spacing'];
```

Comment:
Instead of just excluding these, we could have overrides so we test the schema does validate something reasonable.

Reply:
Have done this, think it's a better approach.